autoload/test/javascript/webdriverio.vim

```diff
@@ -0,0 +1,29 @@
+if !exists('g:test#javascript#webdriverio#file_pattern')
+  let g:test#javascript#webdriverio#file_pattern = '\vtest/specs/.*\.js$'
```

Comment:
Actually, are tests really located in the `test/specs` directory? In any case, it would be better to have an `^` at the beginning of the regex.